langlib/lang.regexp/src/main/java/org/ballerinalang/langlib/regexp/Find.java

```diff
@@ -97,6 +115,15 @@ public static BArray findAllGroups(BRegexpValue regExp, BString str, long startI
         return groupArray;
     }
 
+    private static Matcher getMatcher(BRegexpValue regExp, BString str) {
```

Comment:
Why aren't we calling `RegexUtil.getMatcher()`?

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39444/commits/6143046ac78ebe122ff65b29772bfdb7f62a2554.